test/unit/rubygem_test.rb

```diff
@@ -388,6 +388,14 @@ class RubygemTest < ActiveSupport::TestCase
         assert_equal @rubygem.linkset.bugs, hash["bug_tracker_uri"]
       end
 
+      should "return version documentation url if linkset docs is empty" do
+        @rubygem.linkset.docs = ""
+        @rubygem.save
+        hash = MultiJson.load(@rubygem.to_json)
```

Comment:
Can you just use `JSON` class here instead of `MultiJson`?
I am pretty much sure, we are trying to remove that dependency.